crawl-ref/source/dat/des/arrival/small.des

```diff
@@ -3412,3 +3412,126 @@ MAP
   ...........
    .........
 ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_battle_scene
+TAGS:    arrival no_monster_gen no_trap_gen no_pool_fixup
+ORIENT:  float
+ITEM:    human skeleton / dwarf skeleton , \
+         goblin skeleton / gnoll skeleton / \
+         kobold skeleton / orc skeleton
+SHUFFLE: cx
+SHUFFLE: tvw
+FTILE:   .{det+ = floor_marble
+MAP
+cccc+cccc
+ce..de.ec
+c.e.w..ec
+cd..we..c
+c.d.w.d.c
+c..e..e.c
+ced.{d.ec
+c.d..de.c
+ccccccccc
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_enchanted_forest
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+SHUFFLE: TAG
+FTILE:  .{TAGtF = floor_moss
+MARKER:  F = lua:fog_machine{ \
+             pow_max = 20, delay_min = 20, delay_max = 200, \
+             size = 5, start_clouds = 4, \
+             cloud_type = "black smoke"}
+MAP
+     .tt.t
+  Tt.t.t.tt.F
+ F.tt.A.At.t.t
+ tt.tA...A.tt.
+..t.A..G..Att.t
+tt.A.......A.tt
+.tt..G.{.G....@
+t.tA.......At.t
+t.t.A..G..A.t.t
+ tt.tA...At.tt
+ ..tt.A.At.t.t
+  .F.tt..t.TF
+     t.tt.
+ENDMAP
+
+##############################################################
+NAME:    heliophobic_arrival_ceremonial_halls
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+FTILE:  .+{>G = floor_tomb
```

Comment:
When you use `FTILE` be sure to also think about how the walls look and if you want custom walls, and be sure to set a `COLOUR` directive for console players!

For this vault I recommend `c` stone was with the tomb/ossuary wall tile set and yellow for the `COLOUR`.

It's up to you but it might also be better with the custom floor only inside the starting room, not outside, so it blends into the dungeon.